packages/react-union-scripts/scripts/build.js

```diff
@@ -1,18 +1,48 @@
 const webpack = require('webpack');
 const fse = require('fs-extra');
-const { map, test, complement } = require('ramda');
+const {
+	map,
+	test,
+	complement,
+	flatten,
+	splitEvery,
+	compose,
+	head,
+	prop,
+	invoker,
+	forEach,
+} = require('ramda');
+const path = require('path');
 
 const { getUnionConfig, stats } = require('./lib/utils');
 const configs = require('./webpack.config');
 
+const getClientStatsList = compose(
+	map(head),
+	splitEvery(2),
+	prop('children'),
+	invoker(0, 'toJson')
+);
+
 function build() {
 	return new Promise((resolve, reject) => {
-		webpack(configs).run((err, buildStats) => {
+		const flattenedConfigs = flatten(configs);
+
+		webpack(flattenedConfigs).run((err, buildStats) => {
```

Comment:
This is not very pretty, because this will not allow for applications without SSR to be built. I will figure this out later.